libraries/abstractions/common_io/test/test_iot_uart.c

```diff
@@ -123,8 +131,10 @@ TEST_GROUP( TEST_IOT_UART );
  */
 TEST_SETUP( TEST_IOT_UART )
 {
-    xtestIotUARTSemaphore = xSemaphoreCreateCountingStatic( 10, 0, &xtestIotUARTCompleted );
-    TEST_ASSERT_NOT_EQUAL( NULL, xtestIotUARTSemaphore );
+    xReadCompleteSemaphore = xSemaphoreCreateBinaryStatic( &xReadSemaphoreBuffer );
+    TEST_ASSERT_NOT_EQUAL( NULL, xReadCompleteSemaphore );
```

Comment:
Can the above call fail?

Reply:
Good question. After check FreeRTOS's doc, this call should not fail. So we can remove the TEST_ASSERT_NOT_EQUAL.

https://www.freertos.org/xSemaphoreCreateBinaryStatic.html